DBMS module/System_requirement_analysis.md

```diff
@@ -5,7 +5,7 @@ model is used to form commonality relationships between ER models as a basis for
 integration.
 
 <p align="center">
-<img width="700" height="535" src="images/ER-model.png"> 
+<img width="700" height="535" src="/images/ER-model.png"> 
```

Comment:
@IIITM-Jay Jay you are right... two dots is to change the directory and when Tomy @Antsthebul had already written the name of that directory, it's not required.